guac_core/src/channel_client/types.rs

```diff
@@ -183,7 +204,7 @@ impl UpdateTx {
             false => self.signature_b = Some(*signature),
         }
     }
-    pub fn val_their_signature(&self, is_a: bool) -> bool {
+    pub fn val_their_signature(&self, _is_a: bool) -> bool {
```

Comment:
Calling this `val_*` may be taken as you meaning `value`. Does expending those extra 5 characters hurt this much? Please remember that you're exposing a public function call API here.